miso-web/src/main/webapp/scripts/list_attachment.js

```diff
@@ -44,6 +53,12 @@ ListTarget.attachment = (function() {
               alert('No file selected!');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)